### PR TITLE
test(core.log): fix unit test case

### PR DESCRIPTION
**Pull request by leslie-tsang**
### What this PR does / why we need it:

Currently APISIX `apisix.core.log` test case 6 don't work as expect, this PR fixes it.

Both test case config can pass the prove, which was unexpected.
```
=== TEST 6: print error log with prefix
--- config
    location /t {
        content_by_lua_block {
            local log = require("apisix.core").log.new("test: ")
            log.error("error log")
            log.warn("warn log")
            log.notice("notice log")
            log.info("info log")
            ngx.say("done")
        }
    }
--- log_level: error
--- request
GET /t
--- error_log
error log
--- no_error_log
test: warn log
test: notice log
test: info log
```
```
=== TEST 6: print error log with prefix
--- config
    location /t {
        content_by_lua_block {
            local log = require("apisix.core").log
            log.error("error log")
            log.warn("warn log")
            log.notice("notice log")
            log.info("info log")
            ngx.say("done")
        }
    }
--- log_level: error
--- request
GET /t
--- error_log
error log
--- no_error_log
test: warn log
test: notice log
test: info log
```
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by leslie-tsang** [comment]
@spacewander Could you help me to figure out why this PR can't pass the CI ?
```shell
Notice: ed test 'TEST 6: print error log with prefix - pattern "[notice]" should not match any line in error.log but matches line "2021/12/01 07:15:32 [notice] 30313#30313: init wasm vm: wasmtime" (req 0)
# 2021/12/01 07:15:32 [error] 30315#30315: *23 [lua] content_by_lua(nginx.conf:202):4: raw error log, client: 127.0.0.1, server: localhost, request: "GET /t HTTP/1.1", host: "localhost"
# '
#   at /home/runner/work/apisix/apisix/test-nginx/lib/Test/Nginx/Socket.pm line 1280.
# Looks like you failed 1 test of 70.
```

**Comment by leslie-tsang** [comment]
@membphis ping

**Comment by membphis** [comment]
@leslie-tsang one pr for one thing ^_^

change this PR to smaller

File: apisix/core/log.lua [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
 --
 
-local ngx = ngx
-local ngx_log  = ngx.log
-local require  = require
-local select = select
+local ngx          = ngx
+local ngx_log      = ngx.log
+local require      = require
+local select       = select
```
Comment (spacewander) [inline apisix/core/log.lua:21]: Please avoid changing the irrelevant code

**Review by spacewander — APPROVED** [review]

File: apisix/core/log.lua (hunk 1 above)
Comment (leslie-tsang) [reply apisix/core/log.lua:21]: OK

**Review by moonming — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**